R/dash.R

```diff
@@ -149,26 +157,21 @@ Dash <- R6::R6Class(
       # ------------------------------------------------------------
       router <- routr::RouteStack$new()
 
-      if (!is.null(static_folder)) {
-        local_path <- normalizePath(static_folder, mustWork = TRUE)
-
-        # these should all be directories, right?
-        if (!all(dir_exists(local_path))) {
-          warning(
-            "One or more of the following paths is not a directory, '%s'",
-            paste(static_folder, collapse = "', '"),
+      if (!is.null(private$assets_folder)) {
+        if (!(dir.exists(private$assets_folder))) {
+          warning(sprintf(
+            "The supplied assets folder, '%s' could not be found in the project directory.",
+            private$assets_folder),
```

Comment:
dashPy doesn't look like it has this check. I like the check, but it'll cause problems for people who start an app with the default `assets` here but they don't have (or need) any assets.

Simplest solution I see is to ignore this warning if the folder is the default. It would be even better if we could ignore the error if the user didn't provide a folder name - meaning still give the warning if they provided a value that happens to match the default - but I don't think it would be a good tradeoff to drop the default from the function signature and lose that automatic documentation.

Reply:
Ah, good point. (I should just have a template for your PR reviews where this is the default response.)

So, ultimately what we want is to keep this warning, but
- _don't display it_ when `assets_folder` is set to `assets`, the default, and the folder cannot be found
- _display the warning_ when `assets_folder` is user-supplied, ≠ `assets`, and cannot be found

Did I get that right?